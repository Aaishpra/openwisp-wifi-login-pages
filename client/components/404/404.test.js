import React from "react";
import renderer from "react-test-renderer";

import DoesNotExist from "./404";

describe("<DoesNotExist /> rendering", () => {
  it("should render correctly", () => {
    const component = renderer.create(<DoesNotExist />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
