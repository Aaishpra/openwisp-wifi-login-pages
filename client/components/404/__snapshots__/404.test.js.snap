// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DoesNotExist /> rendering should render correctly 1`] = `
<div
  className="owisp-404-container"
>
  <div
    className="owisp-404-row-1"
  >
    Oops!
  </div>
  <div
    className="owisp-404-row-2"
  >
    404 Not Found
  </div>
  <div
    className="owisp-404-row-3"
  >
    Sorry, an error has occurred, Requested page not found!
  </div>
</div>
`;
